script/playercomponent.js

```diff
@@ -178,7 +180,7 @@ define(
         bubbleBufferingCleared(playbackProperties);
 
         var playbackErrorProperties = createPlaybackErrorProperties(event);
-        raiseError(playbackErrorProperties);
+        raiseError(playbackErrorProperties, false); // TODO: Why was this never set before?
```

Comment:
Any unset parameters are passed as `undefined` so would evaluate as falsey anyway

Reply:
I would vote to keep the parameter for clarity, but remove the TODO.